assets/src/edit-story/karma/fixture/containers/designPanel/sizePosition.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { AbstractPanel } from './abstractPanel';
+
+/**
+ * The video accessibility panel containing inputs for adding poster and title.
+ */
+export class SizePosition extends AbstractPanel {
+  constructor(node, path) {
+    super(node, path);
+  }
+
+  get width() {
+    return this.getByRole('textbox', { name: /Width/i });
+  }
+
+  get height() {
+    return this.getByRole('textbox', { name: /Height/i });
+  }
+
+  get lockAspectRatio() {
+    return this.getByRole('checkbox', { name: /Aspect ratio lock/i });
+  }
```

Comment:
If you use the [`Toggle`](https://github.com/google/web-stories-wp/blob/main/assets/src/edit-story/karma/fixture/containers/common/toggle.js#L28) component, you'll be able to access the element a bit cleaner through the `.button` property (instead of `.nextSibling`), as I've done when [testing the `italic` button](https://github.com/google/web-stories-wp/blob/main/assets/src/edit-story/components/richText/karma/singleSelection.karma.js#L74) in [the `textStyle` panel](https://github.com/google/web-stories-wp/blob/main/assets/src/edit-story/karma/fixture/containers/designPanel/textStyle.js#L40-L46).